include/SDL3/SDL_stdinc.h

```diff
@@ -64,7 +68,7 @@
 #define SDL_INCLUDE_STDBOOL_H
 #endif
 #endif
-#if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) || \
+#if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L && !defined SDL_PLATFORM_NGAGE) || \
```

Comment:
Is ngage compier not gcc-based? What gcc version is that to not have a stdbool.h?

Reply:
That's a long story. In a nutshell: it's a hybrid toolchain. The core is based on a custom-patched GCC 2.9, the C compiler is basically GCC 4.6.4 without any extras (as required since SDL moved on to C99).

Reply:
I added a minimal stdbool.h to the toolchain. Done!